packages/realm/src/schema/types.ts

```diff
@@ -65,7 +65,7 @@ export type RelationshipPropertyTypeName = "object" | "linkingObjects";
  */
 export type UserTypeName = string;
 
-// TODO: Can be removed when ObjectSchemaProperty is removed
+// TODO: Can be removed if/when integration tests no longer use it.
```

Comment:
Please create an issue for it so we don't forget

Reply:
Update: This has now been resolved in this PR 👍